### PR TITLE
Rename Importer to Remote

### PR DESCRIPTION
**Pull request by bmbouter**
https://pulp.plan.io/issues/3496
closes #3496

### CODE REVIEW

**Comment by bmbouter** [comment]
I was able to test against the core PR and pulp_file PR that handle renaming to `Remote`. I verified sync works using the copy/paste instructions from the README.

File: pulp_ansible/app/models.py [hunk 1]
```diff
@@ -62,9 +62,9 @@ class AnsiblePublisher(Publisher):
     TYPE = 'ansible'
 
 
-class AnsibleImporter(Importer):
+class AnsibleRemote(Remote):
     """
-    An Importer for Ansible content
+    A Remote for Ansible content
```
Comment (daviddavis) [inline pulp_ansible/app/models.py]: 👍

File: pulp_ansible/app/tasks/synchronizing.py [hunk 2]
```diff
@@ -110,20 +109,20 @@ def parse_roles(metadata):
...
 
 
-def fetch_roles(importer):
+def fetch_roles(remote):
     """
     Fetch the roles in a remote repository
 
     Args:
-        importer (AnsibleImporter): An importer.
+        remote (AnsibleRemote): An remote.
```
Comment (daviddavis) [inline pulp_ansible/app/tasks/synchronizing.py]: 👎

File: pulp_ansible/app/tasks/synchronizing.py [hunk 3]
```diff
@@ -221,12 +220,12 @@ def find_delta(roles, content, mirror=True):
...
 
 
-def build_additions(importer, roles, delta):
+def build_additions(remote, roles, delta):
     """
     Build the content to be added.
 
     Args:
-        importer (AnsibleImporter): An importer.
+        remote (AnsibleRemote): An remote.
```
Comment (daviddavis) [inline pulp_ansible/app/tasks/synchronizing.py]: 👎

File: pulp_ansible/app/tasks/synchronizing.py [hunk 4]
```diff
@@ -38,51 +37,51 @@
...
     """
-    importer = AnsibleImporter.objects.get(pk=importer_pk)
+    remote = AnsibleRemote.objects.get(pk=remote_pk)
     repository = Repository.objects.get(pk=repository_pk)
     base_version = RepositoryVersion.latest(repository)
 
-    if not importer.feed_url:
-        raise ValueError(_('An importer must have a feed_url specified to synchronize.'))
+    if not remote.feed_url:
+        raise ValueError(_('An remote must have a feed_url specified to synchronize.'))
```
Comment (daviddavis) [inline pulp_ansible/app/tasks/synchronizing.py]: 👎

**Comment by bmbouter** [comment]
All fixed in next push. Thanks @daviddavis :+1: I will merge when core does.

**Comment by bmbouter** [comment]
Core has merged so I'm merging this to stay compatible.

**Pull request merged**